tests/test_mbed_targets.py

```diff
@@ -85,6 +89,31 @@ def test_empty_database_entry(self):
         self.assertEqual("", mbed_target.target_type)
         self.assertEqual("", mbed_target.slug)
 
+    def test_fetch_target_build_attributes(self):
+        mbed_target = _make_mbed_target(board_name="Target_3")
+        expected_result = {
```

Comment:
Another reason why external `subject` is not a good pattern. Ideally the test case should contain all the sample data you need to operate on.

Reply:
715d1b0